cmd/reconstructSeq/reconstructSeq.go

```diff
@@ -3,14 +3,18 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
 	"github.com/vertgenlab/gonomics/expandedTree"
 	"github.com/vertgenlab/gonomics/fasta"
 	"github.com/vertgenlab/gonomics/reconstruct"
 	"log"
 )
 
 func ReconstructSeq(newickInput string, fastaInput string, outputFilename string) {
-	tree := expandedTree.ReadTree(newickInput, fastaInput)
+	tree, err := expandedTree.ReadTree(newickInput, fastaInput)
+	if err != nil {
```

Comment:
The exception.FatalOnErr(err) will do the "if err != nil {" for you, so you can remove this line and just have the single call to FatalOnErr.